### PR TITLE
Integrate particle source

### PR DESCRIPTION
**Pull request by odstrcilt**
Integrate particle source over the timestep of the simulation. It will reduce particle conservation error for noisy or fast varying impurity sources.

### CODE REVIEW

**Comment by odstrcilt** [comment]
@fsciortino  I need the approval of this pull request. I'm not allowed to merge it before approval. All regression tests have passed, so hopefully the installation issues are fixed.

**Comment by fsciortino** [comment]
@odstrcilt I haven't been following this. Is it still relevant? The tests are passed. I can do a quick review if useful.

File: aurora/adas_files.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 )
 
 
-def get_adas_file_loc(filename, filetype="adf11"):
+def get_adas_file_loc(filename, filetype="adf11", tmp_folder=None):
```
Comment (fsciortino) [inline aurora/adas_files.py:35]: @odstrcilt why do you need this `tmp_folder` here?

Comment (odstrcilt) [reply aurora/adas_files.py:35]: I have no idea. I will fix it.

**Comment by odstrcilt** [comment]
@fsciortino ready to be merged

**Pull request closed without merging**